### ISSUE TITLE
False positive on arguments

### ISSUE THREAD

**Issue by sobolevn** [terrencepreilly/darglint#39]
I have this piece of code:

```python
    def __init__(self, node: ErrorNode, text: Optional[str] = None) -> None:
        """
        Creates a new instance of an abstract violation.

        Parameters:
            node: violation was raised by this node. If applied.
            text: extra text to format the final message. If applied.

        """
        self._node = node
        self._text = text
```

When I run `flake8` with `strictness = short` that's what I get:

```
./wemake_python_styleguide/violations/base.py

  77:1     I101  Missing parameter(s) in Docstring: - text
  """
  ^

  77:1     I101  Missing parameter(s) in Docstring: - node
  """
  ^
```

Real source: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/violations/base.py#L76-L86

Looks like a false positive to me. What do you think?

**Comment by terrencepreilly** [comment]
It expects the section to start with either "Args" or "Arguments".  See Google's [style guide](http://google.github.io/styleguide/pyguide.html#doc-function-args), for example.  I personally think it probably should have been "Params"/"Parameters" to begin with, but that's a separate question.

I wouldn't be opposed to adding "Params" or "Parameters" as an alternative, but I would also want to have it raise a style error if they're used in place of "Args", since it's not part of the official style.  That way, a project which uses "Parameters" exclusively should be able to silence that error.

Since the new parser handles these pretty easily, I'll mark this as a feature request and add it as a style error. (Unless I missed somewhere in the docstring style guide/documentation where it mentions or uses this as an alternative.  If you find that, I'll switch this to a bug.)

**Comment by sobolevn** [comment]
@terrencepreilly thanks! This works for me. In the meanwhile I will change it to `Arguments:`, it is even better.

**Comment by sobolevn** [comment]
I would love to see `Arguments:` vs `Args:` option to enforce consistency in naming.

There's a good reference for this tool for js world: https://www.npmjs.com/package/eslint-plugin-jsdoc
